doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -337,6 +337,11 @@ NFC samples
 
   * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
 
+* Added experimental ``llvm`` toolchain support for the ``nrf54l15dk/nrf54l15/cpuapp`` board target to the following samples:
```

Comment:
Move this as the first entry under NFC samples.

Reply:
Moved